Formula/davmail.rb

```diff
@@ -8,7 +8,7 @@ class Davmail < Formula
 
   def install
     libexec.install Dir["*"]
-    bin.write_jar_script libexec/"davmail.jar", "davmail"
+    bin.write_jar_script libexec/"davmail.jar", "davmail", "-Djava.awt.headless=true"
```

Comment:
If this isn't set it will keep a java tab open on the Dock. I can't imagine someone would like that for their personal computer, but maybe I'm in the minority.